apps/app/src/views/components/CopyText.tsx

```diff
@@ -11,7 +11,7 @@ const CopyText = ({ text, size = 'md' }: { text: string; size?: 'xs' | 'sm' | 'm
         whiteSpace={{ base: 'nowrap', sm: 'normal' }}
         overflow={{ base: 'hidden', sm: 'visible' }}
         textOverflow={{ base: 'ellipsis', sm: 'clip' }}
-        fontWeight="medium"
+        fontWeight={size === 'lg' ? 'medium' : undefined}
```

Comment:
what is this for? I don't see us using `CopyText` size `lg` anywhere

Reply:
In a previous PR i started using medium weight copytext for the headers, that made it different weight if it was used below (pharmacy id for example), i added this so it could go back to normal weight below. Quirky, but felt like passing weight in was a bit much.